### PR TITLE
Update appA.go

### PR DESCRIPTION
**Pull request by jgqgoooooooo**
test

### CODE REVIEW

**Comment by jiangguoqing** [comment]
test

**Comment by jgqgoooooooo** [comment]
test

**Review by jiangguoqing — COMMENTED** [review]
test

File: appA.go
Comment (jiangguoqing) [inline appA.go:1]: test it

Comment (jiangguoqing) [reply appA.go:1]: test

**Review by jiangguoqing — COMMENTED** [review]
test

**Review by jiangguoqing — APPROVED** [review]
ok

**Pull request merged**